_posts/2017-03-27-open source update.md

```diff
@@ -0,0 +1,93 @@
+from the wider development community.
+
+Since 2012, we have seen the benefits of an open source by default policy 
+in the following outcomes:
+
+- CFPB’s GitHub organization lists [182 public repositories](https://github.com/cfpb).
+- Projects like eRegulations have been forked and adopted by other government agencies, 
+including the [Bureau of Alcohol, Tobacco, Firearms and Explosives](https://atf-eregs.18f.gov/).
+- We hired a new class of Technology and Innovation Fellows fellows in 2015 and 2016, 
+many of whom cited our open source policy and active GitHub presence 
+as a top reason for applying to and joining CFPB.
+- We don’t just code in the open, we now design in the open too, 
+and [shared our icon library with the Noun Project](https://cfpb.github.io/articles/federal-icons-in-the-open/).
+
+Open source software development has been critical at CFPB since we opened our doors. 
+In April 2012, just nine months after the Bureau opened, 
+we released our [Source Code Policy](https://www.consumerfinance.gov/blog/the-cfpbs-source-code-policy-open-and-shared/), 
+based on the work of the [Department of Defense](http://dodcio.defense.gov/OpenSourceSoftwareFAQ.aspx#OSS_and_DoD_Policy), 
+along with our first two open source projects. 
+Just six days later, 
+[we accepted our first pull request from a member of the public](https://github.com/cfpb/transit_subsidy/pull/1).
+
+Our approach to open source in these early days 
+was to publicly release small bits of source code that had potential for wider application. 
+In November of 2012 and January of 2013, 
+our first class of design and development fellows began working at the Bureau. 
+Two of the projects that were developed by this initial class of fellows, 
+the open data platform [Qu](https://github.com/cfpb/qu) 
+and [eRegulations](https://github.com/cfpb/eRegulations), set a new precedent: 
+they were developed from day one entirely in the open on GitHub. 
+Shortly afterwards, we formed an open source working group, 
+which worked to put procedures in place for open sourcing code that was only available internally. 
+As more code was moved to the public, the team moved towards a policy of open source by default, 
+where source code is developed in private only in certain 
+[well-defined cases](https://github.com/cfpb/source-code-policy/blob/gh-pages/cfpb-source-code-policy.txt#L81-L94), 
+including security risks or when the license is restricted.
+
+With an open source by default policy in place, 
+we’ve been able to collaborate with other government agencies 
+and the public at hackathon events where designers and developers come together 
+to work on open source software:
+
+- CFPB staff have been present at National Day of Civic Hacking events 
+in [2013](https://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking/), 
+[2014](https://www.consumerfinance.gov/blog/we-participated-in-the-national-day-of-civic-hacking-again/),  
```

Comment:
At narrower screen sizes, there's a hard break after the link for the 2014 event. Seems not intentional?

![screen shot 2017-05-22 at 3 51 17 pm](https://cloud.githubusercontent.com/assets/1183435/26325996/50ee15d6-3f07-11e7-8730-b4dc699c39fd.png)